### PR TITLE
Coverage of propose.go made 100%

### LINKED ISSUE #192: Tests for propose

### PR DESCRIPTION
**Pull request by ashish10677**
Fixes #192

### CODE REVIEW

File: cmd/propose.go [hunk 1]
```diff
@@ -140,8 +140,8 @@ func getBiggestInfluenceAndId(client *ethclient.Client, address string, epoch ui
 	return biggestInfluence, biggestInfluenceId, nil
 }
 
-func getIteration(client *ethclient.Client, address string, proposer types.ElectedProposer, utilsStruct UtilsStruct) int {
-	for i := 0; i < 10000000000; i++ {
+func getIteration(client *ethclient.Client, proposer types.ElectedProposer, utilsStruct UtilsStruct) int {
+	for i := 0; i < 10000000; i++ {
```
Comment (SkandaBhat) [inline cmd/propose.go:144]: Why was this number changed?

**Review by SkandaBhat — CHANGES_REQUESTED** [review]

File: cmd/propose.go (hunk 1 above)
Comment (ashish10677) [reply cmd/propose.go:144]: 10000000000 is a very big number, and if the iteration goes almost till that number, it is likely that your block will not be included in the maxAltBlocks. This also slowed down testing if we want to go to the last return after the `for` loop. Hence, I shortened the value.

Comment (SkandaBhat) [reply cmd/propose.go:144]: Needs to be accounted for in a separate issue to make sure we document it

**Comment by ashish10677** [comment]
Will open a new PR wrt change in the iteration max value.

**Pull request closed without merging**